static/img/dart_logo.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
we should pull logo direclty from source and not embed out version of it

Reply:
You're right